rllib/env/multi_agent_episode.py

```diff
@@ -181,7 +181,7 @@ def __init__(
         # via the `module_for()` API even before the agent has entered the episode
         # (and has its SingleAgentEpisode created), we store all aldeary done mappings
         # in this dict here.
-        self._agent_to_module_mapping: Dict[AgentID, ModuleID] = {}
+        self._agent_to_module_mapping: Dict[AgentID, ModuleID] = agent_module_ids or {}
```

Comment:
Great catch!

Reply:
Yeah, I was wondering if this would crash something.